Description: Ensures the compatibility with the latest version of Lucene in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/src/main/org/h2/fulltext/FullTextLucene.java
+++ b/src/main/org/h2/fulltext/FullTextLucene.java
@@ -26,7 +26,7 @@
 import org.apache.lucene.index.IndexWriter;
 import org.apache.lucene.index.IndexWriterConfig;
 import org.apache.lucene.index.Term;
-import org.apache.lucene.queryParser.QueryParser;
+import org.apache.lucene.queryparser.classic.QueryParser;
 import org.apache.lucene.search.IndexSearcher;
 import org.apache.lucene.search.Query;
 import org.apache.lucene.search.ScoreDoc;
@@ -763,7 +763,6 @@
 
         private static void closeSearcher(IndexSearcher searcher) {
             IndexReader indexReader = searcher.getIndexReader();
-            try { searcher.close(); } catch(IOException ignore) {/**/}
             try { indexReader.close(); } catch(IOException ignore) {/**/}
         }
     }
